dlp/createJob.js

```diff
@@ -0,0 +1,77 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// sample-metadata:
+//  title: Create an inspection job
+//  description: Create an inspection job
+//  usage: node createJob.js my-project bucket-url
+function main(projectId, cloudFileUrl) {
+  // [START dlp_create_job]
+  // Imports the Google Cloud Data Loss Prevention library
+  const DLP = require('@google-cloud/dlp');
+
+  // Initialize google DLP Client
+  const dlp = new DLP.DlpServiceClient();
+
+  async function jobsCreate() {
+    // Construct cloud storage configuration
+    const cloudStorageConfig = {
+      cloudStorageOptions: {
+        fileSet: {
+          url: cloudFileUrl,
+        },
+      },
+      timespanConfig: {
+        enableAutoPopulationOfTimespanConfig: true,
+      },
+    };
+
+    // Construct inspect job configuration
+    const inspectJob = {
+      storageConfig: cloudStorageConfig,
+    };
+
+    // Construct inspect configuration
+    const inspectConfig = {
+      infoTypes: [{name: 'PERSON_NAME'}, {name: 'STREET_ADDRESS'}],
```

Comment:
Please use the same info types as existing samples. Please refer to [Java sample](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/51902e4192629ff1184ca7459c5ac597c1c93dfe/dlp/snippets/src/main/java/dlp/snippets/JobsCreate.java#L75).